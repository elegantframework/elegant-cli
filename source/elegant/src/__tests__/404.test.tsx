import ErrorPage from '../pages/404';

describe('The 404 error page', () => {
  it('renders properly', () => {
      const page = ErrorPage();
  
      expect(page).toMatchSnapshot();
  });
})