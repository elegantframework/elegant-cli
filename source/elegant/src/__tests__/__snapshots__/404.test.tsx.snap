// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The 404 error page renders properly 1`] = `
<React.Fragment>
  <div
    className="grid min-h-full place-items-center px-6 pb-24 lg:px-8"
  >
    <div
      className="text-center"
    >
      <p
        className="text-base font-semibold text-primary-500 dark:text-primary-400"
      >
        404
      </p>
      <h1
        className="mt-4 text-3xl font-bold tracking-tight text-slate-900 dark:text-slate-200 sm:text-5xl"
      >
        This page could not be found
      </h1>
      <p
        className="mt-6 text-base leading-7 text-slate-700 dark:text-slate-400"
      >
        Sorry, we couldn't find the page you're looking for.
      </p>
      <div
        className="mt-10 flex items-center justify-center"
      >
        <ForwardRef(LinkComponent)
          className="text-sm font-semibold leading-6 text-primary-500 hover:text-primary-600 dark:text-primary-400 hover:dark:text-primary-500"
          href="/"
        >
          <span
            aria-hidden="true"
            className="mr-1"
          >
            ‚Üê
          </span>
          Back to home
        </ForwardRef(LinkComponent)>
      </div>
    </div>
    <style
      dangerouslySetInnerHTML={
        {
          "__html": "
  body {
    display: flex;
  }
  #__next {
    width: 100%;
    display: flex;
    flex-direction: column;
  }
",
        }
      }
    />
  </div>
</React.Fragment>
`;
